cocotb/handle.py

```diff
@@ -66,11 +70,6 @@ class SimHandleBase(object):
         }
 
     def __init__(self, handle, path):
-        """
-        Args:
-            handle (int): The GPI handle to the simulator object.
-            path (str): Path to this handle, ``None`` if root.
-        """
```

Comment:
Not a huge fan of moving docstrings away from the signature they describe, but maybe there's no way to make it render otherwise.

Reply:
Maybe there is a way to use this docstring inheritance to reduce duplication, but for now I wanted to have the end result to not look bad.

Reply:
Yeah, the issue here is that the subclass repeats the args - removing it there would work too - I'll give that a try in a separate PR.

Reply:
Done in #1229.

Reply:
Rebased to only include the rest of the fixes in this PR.